packages/abstractions/rollup.config.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * -------------------------------------------------------------------------------------------
+ * Copyright (c) Microsoft Corporation.  All Rights Reserved.  Licensed under the MIT License.
+ * See License in the project root for license information.
+ * -------------------------------------------------------------------------------------------
+ */
+
+import resolve from "@rollup/plugin-node-resolve";
+import commonjs from "@rollup/plugin-commonjs";
+
+const config = [{
+    input: ["dist/es/test/browser/index.js"],
```

Comment:
is this rollup configuration only used for tests? (just want to validate the paths)

Reply:
Yeah for now. 

The path here is the index for the test files. The src and tests are rolled up and tested with karma